### DISCUSSION TITLE
This thing is great!

### DISCUSSION THREAD

**Discussion by blugail** [hako-mikan/sd-webui-regional-prompter#153] [Show and tell]
Here's a pic I made with it (I've been using stable diffusion to illustrate a novella). I'd been using "latent couple" for months when it became incompatible with the latest build. But regional prompter is even better! I've been able to make characters that actually look like they are interacting. Nice work!!!! 

![groupshot5](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/136867112/aa0d33c3-09db-481f-acdc-8014b8b01357)

**Comment by andupotorac** [comment]
Care to share your workflow with RP? I'm having such a hard time to get it to simply add a buffer zone to the sides of a photo - the most basic thing.
